examples/remove_widget_from_formDockWidget_example.py

```diff
@@ -0,0 +1,84 @@
+import sys
+
```

Comment:
It's great that the examples show that widgets are sucessfully removed from the dialog and the dock widget.

However, the examples are a bit confusing as they allow you to try to remove widgets that no longer exist, which results in an error being thrown on the command line.
Some possible solutions are greying out buttons and removing items from the dropdown once the widgets they refer to have been deleted, or diverting the errors from the command line into a pop up window on the GUI

Reply:
I went for the option grey out the unused buttons and remove the items from the QComboBox once their associated widgets are removed.